ee/ucp/admin/configure/enable-audit-logging.md

```diff
@@ -1,99 +1,78 @@
+Three audit logging levels are provided to give administrators more control: 
 
-- **None**: audit logging is disabled
-
-- **Metadata**: includes the following:
+- **None:** Audit logging is disabled. 
+- **Metadata:** Includes the following:
     - Method and API endpoint for the request
     - UCP user who made the request
     - Response Status (success or failure)
     - Timestamp of the call
     - Object ID of any created or updated resource (for create or update API 
-    calls). We do not include names of created or updated resources
+    calls). The names of created or updated resources are not included.
     - License Key
     - Remote Address
-
-- **Request**: includes all fields from the Metadata level as well as the 
+- **Request:** Includes all fields from the Metadata level as well as the 
 request payload.
 
 > Note
 >
 > Once UCP audit logging has been enabled, audit logs can be found within the 
 > container logs of the `ucp-controller` container on each UCP manager node. 
-> Please ensure you have a 
-> [logging driver](../../../../config/containers/logging/configure/) 
-> configured appropriately with log rotation set as audit logging can start to 
-> generate a lot of data. 
-
-## Benefits
-
-You can use audit logs to help with the following use cases:
+> Ensure that you have a [logging  driver](../../../../config/containers/logging/configure/) 
+> configured appropriately with log rotation set, as audit logging can start to 
+> generate a large amount of data. 
 
-- **Historical troubleshooting** - Audit logs are helpful in determining a 
-sequence of past events that explain why an issue occurred.
+## Benefits of audit logging
 
-- **Security analysis and auditing** - Security is one of the primary uses for 
-audit logs. A full record of all user interactions with the container 
-infrastructure gives your security team full visibility to questionable or 
-attempted unauthorized accesses.
-
-- **Chargeback** - You can use audit logs and information about the resources to 
-generate chargeback information.
-
-- **Alerting** - If there is a watch on an event stream or a notification 
-created by the event, alerting features can be built on top of event tools that 
-generate alerts for ops teams (PagerDuty, OpsGenie, Slack, or custom solutions).
+Audit logs can be of help with historical troubleshooting, security analysis and auditing, generating chargeback information, and creating useful alerts.
 
 ## Enabling UCP audit logging
 
-UCP audit logging can be enabled via the UCP web user interface, the UCP API or 
-via the UCP configuration file.
+Enable audit logging using either the UCP UI, the UCP API, or the UCP configuration file.
 
-### Enabling UCP audit logging using the web UI
+### Enabling UCP audit logging through the UI
 
-1. Log in to the **UCP** Web User Interface
-2. Navigate to **Admin Settings**
-3. Select **Audit Logs**
+1. Log in to the **UCP** UI.
+2. Navigate to **Admin Settings.**
```

Comment:
Meld Steps 1 and 2 together, as now seen in the following section, **Configure UCP**:

1. Access the UCP UI and browse to the **Admin Settings** page.